cypress.json

```diff
@@ -3,5 +3,6 @@
   "baseUrl": "http://localhost:3000",
   "viewportWidth": 1600,
   "viewportHeight": 1200,
-  "requestTimeout": 10000
+  "requestTimeout": 10000,
+  "video": false
```

Comment:
What does this do?

Reply:
I think it stops the video recording of a test run. I noticed Travis would create mpeg4 after each test. Not sure if somebody looks at those.

Reply:
Mmmm yeah I want this reactivated. I’ll do it in another PR